pkg/tfbridge/provider_test.go

```diff
@@ -1118,8 +1149,8 @@ func TestCheckConfig(t *testing.T) {
 		})
 		require.NoError(t, err)
 		require.Equal(t, 1, len(resp.Failures))
-		require.Equal(t, "could not validate provider configuration: "+
-			"Invalid or unknown key. Examine values at 'explicitprovider.requiredprop'.",
+		require.Equal(t, "could not validate provider configuration: Invalid or unknown key. "+
```

Comment:
Accidental edit, disregard, this stays the same. The explicit provider error should be very rare as it should be a compilation error typically in TypeScript.